src/script/common/helper.cpp

```diff
@@ -39,12 +40,26 @@ template <>
 float LuaHelper::readParam(lua_State *L, int index)
 {
 	lua_Number v = luaL_checknumber(L, index);
-	if (std::isnan(v) && std::isinf(v))
-		throw LuaError("Invalid float value (NaN or infinity)");
+	if (std::isnan(v) || std::isinf(v)) {
+		// TODO turn this into an error
```

Comment:
```suggestion
		// TODO turn this into an error in 5.13.0
```

Reply:
hmm i wasn't sure how long to wait but sure :shrug: